tensorboard/webapp/metrics/store/metrics_store_internal_utils.ts

```diff
@@ -296,35 +296,34 @@ export function buildOrReturnStateWithPinnedCopy(
 }
 
 /**
- * preserving the previous map, which might also contain outdated mapping due to
  * Determines which pinned cards should continue to be present in the metrics
```

Comment:
After submission, if it is feasible, do you mind adding webtests for this functionality. Also perhaps for the use case where the user unpins a card and returns to the experiment later?